nexus-sdk/build.gradle

```diff
@@ -30,3 +31,62 @@ spotless {
 }
 
 classes.dependsOn 'spotlessApply'
+
+ext.getVersionName = { ->
+    def version = System.getenv('NEXUS_SDK_TAG')
+    if (version == null || version.isEmpty()) {
+        return null
+    }
+
+    // Remove "v" prefix if present
+    if (version.startsWith("v")) {
+        version = version.substring(1)
+    }
+    return version
+}
+
+group = 'io.nexusrpc'
+version = getVersionName()
+
+import com.vanniktech.maven.publish.SonatypeHost
+
+mavenPublishing {
```

Comment:
While technically some of this could be in the root project, the plugin didn't like that as much and this is the only project being published at this time. If/when we do add more JARs to this project, we can make some of this logic reusable.